README.md

```diff
@@ -4,10 +4,17 @@
 
 **A Firefox & Chrome extension that displays detailed information about EDAM concepts**
 
-https://addons.mozilla.org/en-US/firefox/addon/edam-popovers/
+Firefox: https://addons.mozilla.org/en-US/firefox/addon/edam-popovers/
+Chrome: https://chrome.google.com/webstore/detail/edam-popovers/amboeicaknkjjpffmdgkopjfljneolca
```

Comment:
Brave: https://chrome.google.com/webstore/detail/edam-popovers/amboeicaknkjjpffmdgkopjfljneolca (the same way as Chrome)